eosetl/enumeration/chain.py

```diff
@@ -0,0 +1,11 @@
+class Chain:
```

Comment:
This file is not needed. @Vasiliy-Bondarenko

Reply:
where are many EOSIO-based chains (TELOS, WORBLI, BOS, etc) so probably will be needed in the future.